Makefile.rules

```diff
@@ -1872,32 +1877,6 @@ check::
 	  $(EchoCmd) No test directory ; \
 	fi
 
-check-lit:: check
```

Comment:
They are effectively dead. They are not documented and nobody is using them AFAIK.